packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -61,6 +62,7 @@ export default class LocusInfo extends EventsScope {
     this.membership = locus.membership || null;
     this.identities = locus.identities || null;
     this.participants = locus.participants || null;
+    this.contorls = locus.controls || null;
```

Comment:
you will need to remove this line as , It wont trigger the initial event when someone joins

Reply:
removed